Assets/MixedRealityToolkit.Tests/PlayModeTests/ManipulationHandlerTests.cs

```diff
@@ -130,15 +130,18 @@ public IEnumerator ManipulationHandlerForceRelease()
 
             // grab the cube - move it to the right 
             var inputSimulationService = PlayModeTestUtilities.GetInputSimulationService();
+
+            // This particular test is sensitive to the number of steps that a hand is moving,
+            // so it's set to 30 to override the default amount.
             int numSteps = 30;
```

Comment:
why not use SlowMoveHand?

Reply:
SlowMoveHand doesn't exist (well, I think it exists in the change that this was based on, but FWIW I wasn't sure I wanted to bring that over. Having another function named SlowMoveHand would create some confusion around which one to use - "do I use fast or slow? maybe I want to use slow so that I can see it happening!" - when really I want to encourage folks to use the fast version always and create an intentional barrier to doing the slow version.